src/twin/mod_test.rs

```diff
@@ -1,62 +1,82 @@
+/*
+    call tests always single threaded since otherwise setting special test env vars clashes:
+
+    'cargo test -- --test-threads=1'
```

Comment:
fwii: we could think about enforcing this option in the build.rs: 
https://github.com/rust-lang/cargo/issues/8430#issuecomment-1547668554